src/main/java/mate/academy/hibernate/relations/service/impl/MovieServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import jakarta.persistence.EntityNotFoundException;
+import mate.academy.hibernate.relations.dao.MovieDao;
 import mate.academy.hibernate.relations.model.Movie;
 import mate.academy.hibernate.relations.service.MovieService;
 
 public class MovieServiceImpl implements MovieService {
+    private final MovieDao movieDao;
+
+    public MovieServiceImpl(MovieDao movieDao) {
+        this.movieDao = movieDao;
+    }
+
     @Override
     public Movie add(Movie movie) {
-        return null;
+        return movieDao.add(movie);
     }
 
     @Override
     public Movie get(Long id) {
-        return null;
+        return movieDao.get(id).orElseThrow(() ->
+                new EntityNotFoundException("Can't get actor by id", new RuntimeException()));
```

Comment:
copy-paste message

Reply:
not fixed

Reply:
The same, you are trying to get Movie, but in messsage you have actor
```suggestion
                new EntityNotFoundException("Can't get movie by id");
```